Calendar/BookingHelpers/BookingTenantSettingsLogic.ps1

```diff
@@ -0,0 +1,156 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function RunTenantTests {
+    [ref]$ErrorMessage = $null
+    [ref]$WriteMessageAlways = $false
+    $TestResult = $true
+
+    Write-DashLineBoxColor "Running Tenant Tests" -Color Blue
+
+    $ErrorMessage = ""
+    $TestResult = CheckBookingsEnabled -errorMessage $ErrorMessage
+    WriteTestResult "Bookings is enabled" -success $TestResult -errorMessage $ErrorMessage
+
+    $TestResult = CheckAcceptedDomainIsManaged -errorMessage $ErrorMessage
+    WriteTestResult "Is Default domain Managed" -success $TestResult -errorMessage $ErrorMessage
+
+    $TestResult = CheckEWSEnabled -errorMessage $ErrorMessage
+    WriteTestResult -title "Check EWS Enabled" -success $TestResult -errorMessage $ErrorMessage -writeMessageAlways $WriteMessageAlways
+
+    $TestResult = CheckEWSAccessPolicy -errorMessage $ErrorMessage -writeMessageAlways $WriteMessageAlways
+    WriteTestResult -title "Check EWS Access Policy" -success $TestResult -errorMessage $ErrorMessage -writeMessageAlways $true
+    #dump allow/block list data below, indented
+    WriteEWSAllowList
+    WriteEWSBlockList
+
+    $TestResult = CheckOWAMailboxPolicyMailboxCreationEnabled -errorMessage $ErrorMessage
+    WriteTestResult "Mailbox Creation is enabled" -success $TestResult -errorMessage $ErrorMessage
+
+    $TestResult = CheckBookingsMBDomain -errorMessage $ErrorMessage
+    WriteTestResult "BookingsMailboxDomain in the correct domain" -success $TestResult -errorMessage $ErrorMessage
+
+    $TestResult = CheckDomainSuffix -errorMessage $ErrorMessage -writeMessageAlways $WriteMessageAlways
+    WriteTestResult "DomainSuffix test for invalid chars" -success $TestResult -errorMessage $ErrorMessage -writeMessageAlways $WriteMessageAlways
+    return
+}
+
+function CheckAcceptedDomainIsManaged {
+    param([ref]$ErrorMessage)
+
+    Write-Verbose "Checking if accepted domain is managed"
+    $AcceptedDomains = $script:TenantSettings.AcceptedDomains
+    if ($null -eq $AcceptedDomains) {
+        $ErrorMessage.Value = "Accepted domains is null."
+        return $false
+    }
+
+    if (@($AcceptedDomains | Where-Object { $_.AuthenticationType -EQ "Managed" -and $_.DomainName -EQ $script:Domain -and $_.Default -EQ $true } ).Count -EQ 0 ) {
```

Comment:
Going to be creating a formatting rule so that operators `-EQ` is lowercase vs uppercase or mixed.

Reply:
modified in the file. Will this rule be created on the CodeFormatter so it gets standardized across all files when using -Save switch ?

Reply:
Correct, if you use the `-Save` switch it will make this change for you.